scripts/kubernetes_test.sh

```diff
@@ -32,8 +32,8 @@ fi
 # Hack to wait for the default service account's creation. https://github.com/kubernetes/kubernetes/issues/66689
 n=0; until ((n >= 60)); do kubectl -n default get serviceaccount default -o name && break; n=$((n + 1)); sleep 1; done; ((n < 60))
 
-kubectl apply -k deploy/kubernetes/overlays/local 
-trap "kubectl delete -k deploy/kubernetes/overlays/local" INT EXIT
+kustomize build deploy/kubernetes/overlays/local/ | kubectl apply -f -
+trap "kustomize build deploy/kubernetes/overlays/local/ | kubectl delete -f -" INT EXIT
```

Comment:
with `delete` you're turning down the environment. is this what you want - in case you had already running env? no biggie, but just checking.

Reply:
That's right - these tests verify full setup / teardown with 1 command. :-)